### PR TITLE
251 new logo and back button on details event

### PR DESCRIPTION
**Pull request by MartinBeaussart**

### CODE REVIEW

File: app/src/main/java/ch/epfl/sdp/ui/main/MainActivity.java [hunk 1]
```diff
@@ -301,4 +302,23 @@ private Intent getUserRefIntent(Class<?> activityClass) {
 
         return intent;
     }
+
+    public void updateToolBarSwipe(boolean goDetails){
```
Comment (JonatanBonjour) [inline app/src/main/java/ch/epfl/sdp/ui/main/MainActivity.java:310]: oublie pas la javadoc !

Comment (MartinBeaussart) [reply app/src/main/java/ch/epfl/sdp/ui/main/MainActivity.java:310]: oui .. :cry:

**Review by JonatanBonjour — APPROVED** [review]
lgtm

**Review by mjmorale — APPROVED** [review]
lgtm

**Pull request merged**